cmd/fetch.go

```diff
@@ -71,3 +82,20 @@ func setSubmissionState(problems []*api.Problem, submissionInfo map[string][]api
 
 	return nil
 }
+
+func fetchAll(trackID string, client *api.Client) []*api.Problem {
```

Comment:
Could we make it so that this function returns the list of problems and the error and `log.Fatal` on the caller if the error is not nil?

Also, what do you think about adding this to the `api.Client` in case we add an API in the future to fetch a list of exercises?